app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,317 @@
+            android:layout_marginStart="12dp"
+            android:gravity="center"
+            android:text="@string/_125_reviews"
+            android:textColor="@color/gray"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toBottomOf="@+id/IV_rating_star_1"
+            app:layout_constraintStart_toEndOf="@id/IV_rating_star_5"
+            app:layout_constraintTop_toTopOf="@+id/IV_rating_star_1" />
+
+        <TextView
+            android:id="@+id/TV_movie_story_line"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:gravity="start"
+            android:text="@string/storyline"
+            android:textColor="@color/white"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_review" />
+
+        <TextView
+            android:id="@+id/TV_movie_announce"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:lineHeight="20dp"
+            android:text="@string/announce"
+            android:textColor="@color/white"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_story_line" />
+
+        <TextView
+            android:id="@+id/TV_movie_cast"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:text="@string/cast"
+            android:textColor="@color/white"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_announce" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="7dp"
+            android:adjustViewBounds="true"
+            android:src="@drawable/image_1"
+            app:layout_constraintEnd_toStartOf="@+id/IV_cast_image_2"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_cast" />
+
+        <ImageView
+            android:id="@+id/IV_cast_image_2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop"
```

Comment:
android:scaleType="centerCrop"
should be added for all images or removed for all

Reply:
android:scaleType="centerCrop"
should be added for all images or removed for all